File: 0485-max-consecutive-ones/0485-max-consecutive-ones.py
class Solution:
    def findMaxConsecutiveOnes(self, arr: List[int]) -> int:
        maxcount=0
        count=0
        for i in range(0,len(arr)):
            if arr[i]==1:
                count+=1
                maxcount=max(maxcount,count)
            else:
                count=0
        return maxcount

File: 0485-max-consecutive-ones/0485-max-consecutive-ones.java
class Solution {

    public int max(int a, int b) {
        if (a >= b)
            return a;
        else
            return b;
    }

    public int findMaxConsecutiveOnes(int[] arr) {
        int maxcount = 0;
        int count = 0;
        for (int i = 0; i < arr.length; i++) {
            if(arr[i]==1){
                count++;
                maxcount=max(maxcount,count);
            }
            else count=0;
        }
        return maxcount;
    }
}